File: feature-indexer/models/feature_type.py
class FeatureType(object):
    def __init__(self, id, title, name, description, keywords, service_id, geometry):
        self.id = id
        self.title = title
        self.name = name
        self.description = description
        self.keywords = keywords
        self.service_id = service_id
        self.geometry = geometry

    def __str__(self):
        return self.__dict__

    def __repr__(self):
        return self

    def serialize(self):
        return {
            'id': self.id,
            'title': self.title,
            'name': self.name,
            'description': self.description,
            'keywords': self.keywords,
            'service_id': self.service_id,
            'geometry': self.geometry,
        }

File: feature-retriever/src/main/java/br/com/helisson/resilientsdi/featureretriever/domain/FeatureType.java
package br.com.helisson.resilientsdi.featureretriever.domain;

import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import java.math.BigDecimal;
import java.util.Date;

@Data
@Entity
public class FeatureType {
    @Id
    private String id;

    @Column
    private String title;

    @Column
    private String name;

    @Column
    private String description;

    @Column
    private String keywords;

    @Column
    private String serviceId;

    @Column
    private Date startDate;

    @Column
    private Date endDate;

    @Column
    private BigDecimal area;

    @Column
    private Integer featuresOfService;

    @Column
    private BigDecimal xMin;

    @Column
    private BigDecimal yMin;

    @Column
    private BigDecimal xMax;

    @Column
    private BigDecimal yMax;
}
